make.bat

```diff
@@ -39,14 +39,27 @@ if "%1" == "help" (
 	echo.  check
 	echo.  serve      to serve devguide on the localhost (8000)
 	goto end
-)
+:help0
 
 if "%1" == "clean" (
 	for /d %%i in (%BUILDDIR%\*) do rmdir /q /s %%i
 	del /q /s %BUILDDIR%\*
 	goto end
 )
 
+rem If we get this far, we're going to use the SPHINXBUILD command
+if not defined SPHINXBUILD (
+	rem If it is not defined, we build in a virtual environment
+	if not exist venv (
```

Comment:
This needs some protection against running the script from somewhere other than the dir holding the script. I think `if not exist %~dp0venv` should do it, but make sure I remembered the correct flags.

Reply:
Sorry, missed this comment the first look. I'll think about this. First thought is that it didn't have it before.

Second is that it might be useful and another thing I could take from the CPython docs build. I guess it would pushd to the right place ... https://github.com/python/cpython/blob/8d59aca4a953b097a9b02b0ecafef840e4ac5855/Doc/make.bat#L4

Reply:
To clarify, I think the idiom in that example would make it run correctly whatever the current directory, but I'll check that (and figure out whether `popd` is needed).